consul/tests/test_service.sls

```diff
@@ -1,5 +1,5 @@
 consul_service_running:
```

Comment:
It might be useful to prefix this with `test_` so that in the output we know what the purpose of the state is.